quiche/src/lib.rs

```diff
@@ -2815,11 +2846,25 @@ impl Connection {
             });
         }
 
+        // Foll. flag used to upgrade datagram size, if probe successful
+        let mut pmtud_probe = false;
+
         // Process acked frames. Note that several packets from several paths
         // might have been acked by the received packet.
         for (_, p) in self.paths.iter_mut() {
             for acked in p.recovery.acked[epoch].drain(..) {
                 match acked {
+                    frame::Frame::Ping {
+                        mtu_probe: Some(mtu_probe),
+                    } => {
+                        let pmtud_next = p.pmtud.get_current();
+                        p.pmtud.set_current(cmp::max(pmtud_next, mtu_probe));
+                        // Stop sending path MTU probes after successful
+                        // probe
+                        trace!("PMTUD: Acked");
+                        p.pmtud.should_probe(false);
+                        pmtud_probe = true;
+                    },
```

Comment:
Would it make sense to write a qlog event at this point? - dependent on https://github.com/cloudflare/quiche/pull/1641